nodeJS/final_project/project_odin_book.md

```diff
@@ -1,42 +1,50 @@
+### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Users must sign in to see anything except the sign in page.
-2. Users should be able to sign in using their real facebook details.  This is fairly easily accomplished using PassportJS, and you should be able to use the knowledge you already have to figure it out [from the documentation](http://www.passportjs.org/docs/facebook/).
-3. Users can send friend requests to other users.
-4. A user must accept the friend request to become friends.
-5. Users can create posts. (begin with text only)
-6. Users can like posts.
-7. Users can comment on posts.
-8. Posts should always display with the post content, author, comments and likes.
-9. Treat the Posts index page like the real Facebook's "Timeline" feature -- show all the recent posts from the current user and users they are friends with.
-10. Users can create  Profile with a photo (you can get this from the real facebook when you sign in using passport)
-11. The User Show page contains their profile information, profile photo and posts.
-12. The Users Index page lists all users and buttons for sending friend requests to those who are not already friends or who don't already have a pending request.
-13. Deploy your app to a host provider of your choice!
 
-#### Extra credit
+Build a social media site! You'll build a large portion of the core user functionality of your chosen site in this project. You don't have to worry about some of the more flashy front-end stuff unless you want to, but you don't need it to get a nice user experience.
 
-1. Make posts also allow images (either just via a url, or by uploading one.)
-2. Allow Users to upload and update their own profile photo.
-3. Create a guest sign-in functionality that allows visitors to bypass the login screen without creating an account or supplying credentials. This is especially useful if you are planning on putting this project on your résumé - most recruiters, hiring managers, etc. will not take the time to create an account. This feature will give them an opportunity to look at your hard work without going through a tedious sign-up process.
-4. Make it pretty!
+This project will give you the chance to take a relatively high-level set of requirements and turn it into a functioning website. You'll need to do some of your own research and read the documentation for a few of the modules you'll be using in this project.
+
+The following requirements are a very global list of features your app should have. Because of the open-ended nature of this project, it's possible that not all of them may apply to your chosen site, and that there might be core features of your site that aren't mentioned here. Keep in mind that the overall goal is to clone your site, not to tick off a list of requirements.
```

Comment:
The last sentence feels redundant, thoughts on removing it?
```suggestion
The following requirements are a very global list of features your app should have. Because of the open-ended nature of this project, it's possible that not all of them may apply to your chosen site, and that there might be core features of your site that aren't mentioned here.
```